common/app/templates/headerInlineJS/bootCurl.scala.js

```diff
@@ -99,6 +99,11 @@ require([
         }
     };
 
+    // IE8 and below use attachEvent
+    if (!window.addEventListener) {
+        window.addEventListener = window.attachEvent;
+    }
+
     // Report unhandled promise rejections
     // https://github.com/cujojs/when/blob/master/docs/debug-api.md#browser-window-events
```

Comment:
Inside Scala templates if you do comments like this then they get stripped out at compile time and don't end up in the page source...

`@*  this is a comment  *@`